packages/oidc_core/lib/src/managers/user_manager_base.dart

```diff
@@ -784,15 +784,6 @@ abstract class OidcUserManagerBase {
     if (user == null) {
       tokenEventsManager.unload();
     } else {
-      if (!discoveryDocument.grantTypesSupportedOrDefault
```

Comment:
This is removed cause `handleTokenExpiring` should be called even if there is no refreshToken...